### DISCUSSION TITLE
Pack Education

### DISCUSSION THREAD

**Discussion by AzaleeX** [community/community#72873] [GitHub Education]
### Hi everyone,

My name is Azalee and I recently applied for the GitHub Education Pack. However, I'm facing a problem with my student pack approval. The application requires a Date school ID but I don't know what that means, but I don't have a school email address. I tried submitting my schooling certification instead, but that didn't solve the problem. I'm sure the image I uploaded is correct, but the approval doesn't seem to happen automatically.

I'm very keen to use the resources offered by the GitHub Education Pack for my academic projects, so I'm very keen to resolve this issue as soon as possible. Would it be possible to check my request manually?

Thank you for your assistance and I look forward to hearing from you. If you need any further information from me, please let me know.

Best regards,
Azalee

**Comment by 4lexxe** [comment]
Hi Azalee! To access the benefits of the GitHub Education Developer Pack, it is essential to have an institutional email address provided by your university or school. Additionally, it's important that your educational institution is affiliated with this program for you to be eligible. When you try to sign up without these requirements, the system will not allow you to proceed. Once you have all the necessary details, you can apply and gain access to these benefits, which typically takes around a week to be approved. Therefore, your request for these benefits might not be considered. I suggest you go to your educational institution and inquire if you can have an institutional email, but what matters there is if your school/university is truly affiliated with this program. In case it's not, consider changing your affiliation. Haha, ok no xD
